core/src/main/java/org/elasticsearch/action/update/UpdateRequestBuilder.java

```diff
@@ -122,10 +121,16 @@ public UpdateRequestBuilder setVersionType(VersionType versionType) {
     }
 
     /**
-     * Sets the consistency level of write. Defaults to {@link org.elasticsearch.action.WriteConsistencyLevel#DEFAULT}
+     * Sets the number of shard copies that must be active for the write to proceed.
+     * Defaults to 1, which means only the primary is required to be active.  Set this
+     * value to -1 to ensure all shards (primary and all replicas) are active before
+     * proceeding with the write.  A value of 0 implies the value is not set, so the
+     * default will be used.
+     *
+     * @param waitOnActiveShards number of active shard copies to wait on, must be between 0 and numShardCopies, or -1 for all.
      */
-    public UpdateRequestBuilder setConsistencyLevel(WriteConsistencyLevel consistencyLevel) {
-        request.consistencyLevel(consistencyLevel);
+    public UpdateRequestBuilder setWaitOnActiveShards(int waitOnActiveShards) {
```

Comment:
We also have `ReplicationRequestBuilder`. No need to add this to individual builders.